rtpcodec.go

```diff
@@ -123,3 +124,15 @@ func codecParametersFuzzySearch(needle RTPCodecParameters, haystack []RTPCodecPa
 
 	return RTPCodecParameters{}, codecMatchNone
 }
+
+// Given a CodecParameters find the RTX CodecParameters if one exists
+func findRTXCodecParameters(needle RTPCodecParameters, haystack []RTPCodecParameters) (RTPCodecParameters, bool) {
```

Comment:
The `apt` parameter is specifix to RTX (RFC 4588).  I think this would be very slightly clearer if the *needle* were just the PayloadType, we don't need the full data here.

Reply:
fixed! Made it so PayloadType is passed directly